utils/web_scraper.py

```
@@ -1880,8 +1831,740 @@ def format_window_context_content(content: Dict[str, Any]) -> str:
+    except Exception as e: 
+        logger.error(f"Twitter API 获取失败: {e}")
+        return {'success': False, 'error': str(e)}
+
+async def fetch_personal_dynamics(limit: int = 10) -> Dict[str, Any]:
+    """
+    独立获取全平台个人登录态下的订阅/关注动态
+    """
+    try:
+        china_region = is_china_region()
+        if china_region:
+            logger.info("检测到中文区域，获取B站和微博个人动态")
```

Ruff (0.15.1)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)